azure-keyvault-cryptography/pom.xml

```diff
@@ -51,18 +49,33 @@
         <dependency>
           <groupId>org.bouncycastle</groupId>
           <artifactId>bcprov-jdk15on</artifactId>
-          <scope>test</scope>
-          <version>1.54</version>
+          <version>1.59</version>
         </dependency>
         <dependency>
             <groupId>com.microsoft.azure</groupId>
-            <artifactId>azure-keyvault-core</artifactId>
-            <version>1.1-alpha-2</version>
+            <artifactId>azure-keyvault</artifactId>
+            <version>1.1-beta-1</version>
         </dependency>
```

Comment:
adding a dependency on the client  module (azure-keyvault) seems a bit over the top just to pull in JsonWebKeyCurveName which defines 4 constant strings.  I'm not generally a fan of duplicating constants but in this case I think it might be warranted.